dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -4,6 +4,34 @@
 
 set -euo pipefail
 
+# Load dracut libraries. Using getargbool() and getargs() from
+# dracut-lib and ip_to_var() from net-lib
+load_dracut_libs() {
+    # dracut is not friendly to set -eu
+    set +euo pipefail
+    type getargbool &>/dev/null || . /lib/dracut-lib.sh
+    type ip_to_var &>/dev/null  || . /lib/net-lib.sh
```

Comment:
Oh, it's totally fine IMO to just source these upfront and only doing our `set -euo pipefail` afterwards. This works fine too though of course.